src/plugins/amqplib.js

```diff
@@ -98,12 +98,21 @@ function addTags (channel, tracer, config, span, method, fields) {
     'destination'
   ]
 
+  let destinationName = fields.exchange
+  if (destinationName === undefined) {
+    destinationName = fields.queue
+  }
+
   span.addTags({
     'service.name': config.service || `${tracer._service}-amqp`,
     'resource.name': getResourceName(method, fields),
     'component': 'amqplib'
   })
 
+  if (destinationName) {
```

Comment:
nit: why is this separate from the var definition?

Reply:
Because I want the tag to be set after the the tags are set.